src/OrchardCore.Modules/OrchardCore.OpenId/Configuration/OpenIdClientConfiguration.cs

```diff
@@ -68,6 +68,7 @@ public void Configure(string name, OpenIdConnectOptions options)
             options.GetClaimsFromUserInfoEndpoint = true;
             options.ResponseMode = settings.ResponseMode;
             options.ResponseType = settings.ResponseType;
+            options.SaveTokens = settings.StoreExternalTokens;
```

Comment:
Where is it used?

Reply:
Setting SaveTokens to true, the authentication tokens are preserved in the ExternalLoginInfo in the AccountController.